src/components/modalDialogs/NotificationFilters.vue

```diff
@@ -27,7 +27,7 @@
           v-on:input="(vals) => updateEventStates(vals)"
           switches
           stacked
-          class="settings-group"
+          class="settings-group fissureSettingGroup notificationGroup"
```

Comment:
Classes should be dash separated instead of camel, please... I'm sure there's some existing inconsistency from our own code, but please keep it consistent with bootstrap's convention

Reply:
Sure